File: project 1 - math/js/change-degree.js
  
        "use strict";  
        let choice = prompt("Enter 'd' for degree to radians or 'r' for radians to degree").toLowerCase();  
        
        if (choice === "d") {  
            let degreeInput = Number(prompt('Enter your degree'));  
            let radian = degreeInput * (Math.PI / 180);  
            alert("The result in radians is: " + radian);  
        }   
        else if (choice === "r") {  
            let radianInput = Number(prompt('Enter your radians'));   
            let degree = radianInput * (180 / Math.PI);  
            alert("The result in degrees is: " + degree);  
        }   
        else {  
            alert("Invalid input");  
        }  
   

File: project 1 - math/python/change-degree.py
import math
choose = input("degree = d and radians = r")

if choose == "d":
    prompt = float(input("enter your degree"))
    result =   prompt * (math.pi / 180)
    print(f"your radians = {result}")
elif choose == "r":
    prompt = float(input("enter your radians"))
    result = prompt * (180 / math.pi)
    print(f"your degree = {result}")  
else:
    print("invalid input")

